website_v2/gcloud-powershell-website-v2/www/index.html

```diff
@@ -0,0 +1,70 @@
+﻿<!DOCTYPE html>
+<html class="google-js" lang="en" ng-app="powershellSite">
+<head>
+  <meta charset="utf-8">
+  <script>
+      (function(H){H.className=H.className.replace(/\bgoogle\b/,'google-js')})(document.documentElement)
+  </script>
+  <meta content="initial-scale=1, minimum-scale=1, width=device-width" name="viewport">
+  <meta name="description" content="">
+  <base href="/">
+  <link rel="shortcut icon" href="https://cloud.google.com/images/gcp-favicon.ico">
+
+  <title>Google Cloud PowerShell Cmdlet Reference</title>
+  <script src="//www.google.com/js/google.js"></script>
+  <link rel="canonical" href="//googlecloudplatform.github.io/google-cloud-powershell/cmdlets/">
+  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&amp;lang=en">
+  <link rel="stylesheet" href="static/css/maia.css" />
+  <link rel="stylesheet" href="static/css/style.css" />
+  <script src="static/scripts/angular.min.js"></script>
+  <script src="static/scripts/angular-route.js"></script>
```

Comment:
Why not bring in angular-route.min.js? Do we need the non-minified version for some reason?